code/solutions/understanding-changesets/bucket.yaml

```diff
@@ -1,6 +1,6 @@
 AWSTemplateFormatVersion: "2010-09-09"
 
-Description: AWS CloudFormation workshop - Change sets - S3 bucket (uksb-1q9p31idr).
+Description: AWS CloudFormation workshop - Change sets - S3 bucket (uksb-1q9p31idr) (tag:change-sets).
```

Comment:
The parent directory doesn't have a hyphen in `changesets`. I don't mind which form you use, but it would be best to keep it consistent

Reply:
Good catch. Changed to proposed tag.